docs/docusaurus/docs/guides/expectations/how_to_edit_an_existing_expectationsuite.md

```diff
@@ -0,0 +1,154 @@
+```
+
+Under the hood, the Validator will be creating and updating an Expectation Suite, which we can view next. 
+
+### 4. View the Expectations in the Expectation Suite
+
+There are a number of different ways that this can be done, with one way being using the `show_expectations_by_expectation_type()` function, which will use `prettyprint` to print the Suite to the console in a way that can be easily visualized. 
+
+First load the `ExpectationSuite` from the `Validator`: 
+
+```python name="tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite get_suite"
+```
+
+Now use the `show_expectations_by_expectation_type()` to print the Suite to console or Jupyter Notebook.
+
+```python name="tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite show_suite"
+```
+
+Your output will look something similar to this: 
+
+```bash 
+[ { 'expect_column_values_to_be_between': { 'auto': True,
+                                            'column': 'passenger_count',
+                                            'domain': 'column',
+                                            'max_value': 6,
+                                            'min_value': 1,
+                                            'mostly': 1.0,
+                                            'strict_max': False,
+                                            'strict_min': False}},
+  { 'expect_column_values_to_not_be_null': { 'column': 'pickup_datetime',
+                                             'domain': 'column'}}]
+```
+
+### 5. Instantiate ExpectationConfiguration 
+
+From the Expectation Suite, you will be able to create an ExpecationConfiguration object using the output from `show_expectations_by_expectation_type()` Here is the example output of the first Expectation in our suite.
+
+It runs the `expect_column_values_to_be_between` Expectation on the `passenger_count` column and expects and min and max values to be `1` and `6` respectively. 
+
+```python name="tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite example_dict_1"
+```
+
+Here is the same configuration, but this time as a `ExpectationConfiguration` object.  
+
+```python name="tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite example_configuration_1"
+```
+
+### 6. Update Configuration and ExpectationSuite
+
+Let's say that you are interested in adjusting the `max_value` of the Expectation to be `4` instead of `6`. Then you could create a new `ExpectationConfiguration` with the new value: 
+
+```python name="tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite updated_configuration"
+```
+
+And update the ExpectationSuite by calling `add_expectation()`. The `add_expectation()` function will perform an 'upsert' into the `ExpectationSuite`, meaning it will update an existing Expectation if it already exists, or add a new one if it doesn't. 
```

Comment:
Does is only upset if the expectations have the same title/name?

Reply:
By default it is going to be the same domain (column name in this case) and Expectation name.